src/uni/SMU.ts

```diff
@@ -87,10 +105,10 @@ function generateTerm(start: Moment, label: string, vacationWeekCount: number) {
 function getVacationWeekCount(termNum: number) {
   switch (termNum) {
     case 1: {
-      return 6;
+      return 5;
```

Comment:
should this remain as 6?

Reply:
Nah, this one correct. SMU 1st week of vacation is technically exam week. The exams only end on Friday.